includes/admin/admin-bar.php

```diff
@@ -19,48 +19,126 @@
  */
 function edd_maybe_add_store_mode_admin_bar_menu( $wp_admin_bar ) {
 
-	// Bail if no admin bar
+	// Bail if no admin bar.
 	if ( empty( $wp_admin_bar ) ) {
 		return;
 	}
 
-	// Bail if user cannot manage shop settings
+	// Bail if user cannot manage shop settings.
 	if ( ! current_user_can( 'manage_shop_settings' ) ) {
 		return;
 	}
 
-	// String
+	$indicator     = '';
+	$notifications = EDD()->notifications->countActiveNotifications();
+
+	if ( $notifications ) {
+		$count     = $notifications < 10 ? $notifications : '!';
+		$indicator = ' <div class="wp-core-ui wp-ui-notification edd-menu-notification-counter">' . $count . '</div>';
```

Comment:
What would you think of only showing the admin bar notifications indicator if we are not on a screen with the EDD header active? Having both red circles right next to each other (this may be due to my screen size) feels like a lot:

![image](https://user-images.githubusercontent.com/4853066/175281083-f21718c8-107c-4341-b566-a13387fc7268.png)

If that's just me, though, it's fine and I can live with both present.

Reply:
@robincornett  I think this is just the nature of the current environment you're in, I'm on a smaller screen and with less menu bar items they aren't as close.

keep in mind that you have the EDD Dev tools installed, as well as Query Monitor, so in a production environment, your EDD menu item would be all the way to the left in your screenshot, b/c the QM fields, Blog #, branch identifier and settings dropdown will all be gone.